react/javascript/tile-visualization/README.md

```diff
@@ -0,0 +1,184 @@
+# Looker Tile Visualization Extension Demo (React & JavaScript)
+
+Please note, this type of extension requires Looker 24.0 and above.
+
+This example demonstrates an extension that can be mounted as a tile in a dashboard. The extension has two modes:
+
+- Visualization: the extension behaves similarly to a custom visualization. The tile extension can be used in an explore and saved to a dashboard. The data is pushed to the extension via the explore or the dashboard.
+- Tile: the extension is responsible for retrieving its own data. The extension can be added to a dashboard using the `Add` button when a dashboard is in edit mode.
+
+## Getting Started for Development
+
+1. Clone the Looker extension examples repo.
+
+   ```
+   # cd ~/ Optional, your user directory is usually a good place to git clone to.
+   git clone git@github.com:looker-open-source/extension-examples.git
+   ```
+
+2. Navigate (`cd`) to the example directory on your system
+
+   ```
+   cd extension-examples/react/javascript/tile-visualization
+   ```
+
+3. Install the dependencies with `npm`.
+
+   ```
+   npm install
+   ```
+
+4. Start the development server
+
+   ```
+   npm run develop
+   ```
+
+   The extension is now running and serving the JavaScript locally at https://localhost:8080/bundle.js.
+
+5. Log in to Looker and create a new project.
+
+   This is found under **Develop** => **Manage LookML Projects** => **New LookML Project**.
+
+   Select "Blank Project" as your "Starting Point". This will create a new project with no files.
+
+   1. The extension folder has a `manifest.lkml` file.
+
+   Either drag & upload this file into your Looker project, or create a `manifest.lkml` with the same content. Change the `id`, `label`, or `url` as needed.
+
+   ```
```

Comment:
this is handy but you could just use the link to the manifest file in the folder to avoid sync/update issues?

Reply:
I'm going to leave for now as it is consistent with the rest of the examples